<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>770</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>770</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>770</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="ControlsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>731</height>
     </rect>
    </property>
    <property name="title">
     <string>Controls</string>
    </property>
    <widget class="QPushButton" name="btnNextGeneration">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>161</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Next Generation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAction">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGetRandomSpace">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>161</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Random space</string>
     </property>
    </widget>
    <widget class="QSlider" name="sldProbQ0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sldProbQ1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sldProbQ2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="txtProbQ1">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>220</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Probability Q1: 0.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtProbQ2">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>280</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Probability Q2: 0.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtProbQ0">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Probability Q0: 0.00</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkUseToroid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Use toroid space</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>380</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Born condition</string>
     </property>
     <widget class="QSpinBox" name="spbB1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spbB2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>440</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Survival condition</string>
     </property>
     <widget class="QSpinBox" name="spbS1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spbS2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>500</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Upgrade condition</string>
     </property>
     <widget class="QSpinBox" name="spbU1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spbU2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>560</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Remain condition</string>
     </property>
     <widget class="QSpinBox" name="spbR1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spbR2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>620</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Dimensions width and height</string>
     </property>
     <widget class="QSpinBox" name="spbDimensions">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>680</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Cells size (pixels)</string>
     </property>
     <widget class="QSpinBox" name="spbCellSize">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>761</width>
      <height>721</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>761</width>
      <height>721</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>761</width>
      <height>721</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>751</width>
       <height>681</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>728</width>
        <height>658</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="txtGeneration">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>690</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Generation:</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtnQ0Cells">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>690</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Q0 cells: </string>
     </property>
    </widget>
    <widget class="QLabel" name="txtnQ1Cells">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>690</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Q1 cells: </string>
     </property>
    </widget>
    <widget class="QLabel" name="txtnQ2Cells">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>690</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Q2 cells: </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
